src/pages/SubmissionDetails.test.js

```diff
@@ -102,19 +102,6 @@ describe('Submission Details page', () => {
     Object.defineProperty(HTMLElement.prototype, 'offsetWidth', { configurable: true, value: 800 })
   })
 
-  beforeEach(() => {
-    const getRunsMethod = jest.fn(() => Promise.resolve(runsData))
```

Comment:
this change has implications throughout the test, but is necessary for testing the case where `runsData` should be empty.

Reply:
What happens if you keep this here, but override it's implementation locally inside the test when you want to return empty `runsData`?

Reply:
you know what, that seems to work just fine. good catch.

given that, do you think I should restore the `beforeEach`? I don't feel strongly about it, but I kind of like having each test set itself up explicitly (I'm less particular about DRY when it comes to tests, personally)

Reply:
I think it would make sense to restore `beforeEach` because we use the same setup for almost all tests (and explicitly change it inside one test). That being said I don't have any strong opinion either way so I would be happy whichever way you decide to implement it.